### PR TITLE
Photo page updates

### PR DESCRIPTION
**Pull request by jtessler**
Update the style of the photos page, which is now consistent with the site-wide themes. Removed link to "All Photos" for obvious security reasons. Extended the contentArea to fill the entire column.

### CODE REVIEW

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -182,8 +182,7 @@ height: 80%; /*Resize the profile-picture here, picture will take up all the spa
 
 /** contentArea **/
 #contentArea {
-
-  /*border-left: 1px solid #dddddd;*/    
+  width: 100%;
```
Comment (jtessler) [inline public/stylesheets/main.css:185]: This looks fine in FF and Chrome on linux. If it looks bad on Windows/Mac, we can revert this change.

File: app/views/tags/nav.html [hunk 2]
```diff
@@ -34,11 +34,9 @@
...
 </div>
 
-  <div class="lhs-navBoxHeader">PHOTOS</div>  
-   <div class="lhs-navBoxBody">  
-  <a href="@{Photos.photos(_user.id)}">My Photos</a>
-  <a href="@{Photos.photos()}">All Photos</a>
-  <br />
+  <div class="lhs-navBoxHeader">PHOTOS</div>
+  <div class="lhs-navBoxBody">
+    <a href="@{Photos.photos(_user.id)}">My Photos</a>
```
Comment (jtessler) [inline app/views/tags/nav.html]: Why do we have two identical nav templates?

File: app/views/Photos/photos.html [hunk 3]
```diff
@@ -1,41 +1,35 @@
...
-					type="submit" name="Set Background Photo" value="Set as background" />
-					#{/form} #{/if}</li> #{/list}
-			</ul>
-		</div>
-		<div id="content-right"></div>
+  <h2>Your photos</h2>
+  <hr />
 
-	</div>
+  <ul class="photo-list">
```
Comment (amshali) [inline app/views/Photos/photos.html:29]: Looks good

File: app/views/tags/lhsNav.html [hunk 4]
```diff
@@ -32,8 +32,7 @@
 
 <div class="lhs-navBoxHeader">PHOTOS</div>
 <div class="lhs-navBoxBody">
-	<a href="@{Photos.photos(_user.id)}">My Photos</a> <a
-		href="@{Photos.photos()}">All Photos</a> <br />
+	<a href="@{Photos.photos(_user.id)}">My Photos</a>
```
Comment (amshali) [inline app/views/tags/lhsNav.html]: Why do you send the user id? Can't you just get the user by calling user() inside the controller?

File: app/views/tags/nav.html (hunk 2 above)
Comment (amshali) [inline app/views/tags/nav.html]: user id issue mentioned above.

File: app/views/tags/photo.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<li>
+  <img class="thumbnail" src=@{Photos.getPhoto(_photo.id)} />
+  #{if _currentUser.equals(_photo.owner)}
+    <br />
+    <a href="@{Photos.removePhoto(_photo.id)}"
```
Comment (amshali) [inline app/views/tags/photo.html:5]: Do you also check this inside the removePhoto?

File: app/views/tags/lhsNav.html (hunk 4 above)
Comment (jtessler) [inline app/views/tags/lhsNav.html]: You're right, I should modify the controller. Currently, if you do not pass any parameters, it returns ALL photos. I'll update that now.

File: app/views/tags/photo.html (hunk 5 above)
Comment (jtessler) [inline app/views/tags/photo.html:5]: Yes, `removePhoto()` only deletes if the current user owns the image.

**Comment by amshali** [comment]
I am gonna merge for now. LGTM.
